src/blueapi/service/subprocess_handler.py

```diff
@@ -113,6 +119,11 @@ def get_task_by_id(self, task_id: str) -> TrackableTask | None:
     def initialized(self) -> bool:
         return self._initialized
 
+    @property
+    # This method is not currently on ABC
+    def error(self) -> bool:
```

Comment:
Should we just have the fact that there is/isn't an error or actually report an error message via the API?

Reply:
Possibly, although I don't know how straightforward it is to get a meaningful message. The exception message might not give enough context, and the whole exception stack trace is a bit unwieldy.

Reply:
Currently, when a plan fails, we pass just the exception message and expect the user to look at the logs if they require more info. Could do the same here?

Reply:
Ok, yeah will look at doing that.